service/src/test/java/com/ge/predix/controller/test/AttributeConnectorControllerIT.java

```diff
@@ -86,6 +86,15 @@ public void setup() {
     }
 
     @Test(dataProvider = "requestUrlProvider")
+    public void attributeConnectorZoneDoesNotExistException(final String endpointUrl) throws Exception {
+        createZone1AndAssert();
```

Comment:
No need to setup a zone if we are testing "zone does not exist" use case.

Reply:
I reverted my changes as there is a test already checking for zoneDoesnotExist with test name TestZoneDoesNotExist